src/ParametersInterface.php

```diff
@@ -57,24 +62,20 @@ public function toArray();
      *
      * Allow serialization to query format; e.g., for PUT or POST requests
      *
-     * @return mixed
+     * @return string
      */
     public function toString();
 
     /**
-     * Get
-     *
-     * @param string $name
-     * @param mixed|null $default
+     * @param TKey $name
+     * @param TValue|null $default
      * @return mixed
```

Comment:
shouldn't this also `@return TValue|null`?

Reply:
:shrug: no idea about the current implementations